src/HotChocolate/AspNetCore/src/Transport.Abstractions/Serialization/Utf8JsonWriterHelper.cs

```diff
@@ -206,14 +206,7 @@ public static void WriteOperationRequest(Utf8JsonWriter writer, OperationRequest
 
         for (var i = 0; i < list.Count; i++)
         {
-            var element = list[i];
-
-            if (element is null)
-            {
-                continue;
-            }
-
-            WriteFieldValue(writer, element);
+            WriteFieldValue(writer, list[i]);
```

Comment:
We need the null values also in arrays

Reply:
Ah ... i know why i did that ... if null i wanted to short-circur

writer.WriteNullValue()

in the null check

Reply:
uhh ... there are a couple of bugs in there .... I will need to do a separate PR

Reply:
damn it michael ;)

Reply:
in the Utf8JsonWriter?